<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brightness Control v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="br_mapping_groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>564</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Auto Light Sensor/Brightness Level ( Default level - 2, 5, 5, 7, 7, 9, 9, 9)</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="holdtime_label">
         <property name="text">
          <string>Brightness Change Holdtime ( Default: 10)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="holdtime">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>30</number>
         </property>
         <property name="value">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="l1">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="br_l1">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l2">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="br_l2">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l3">
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="br_l3">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l4">
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="br_l4">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l5">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="br_l5">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l6">
         <property name="text">
          <string>6</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="br_l6">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l7">
         <property name="text">
          <string>7</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="br_l7">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="l8">
         <property name="text">
          <string>8</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="br_l8">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>10</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="auto_Apply">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>448</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>31</y>
      <width>561</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="autoBREN_checkBox">
       <property name="text">
        <string>Auto Brightness Control</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>28</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="frmVersion_label">
       <property name="text">
        <string>Firmware version: V1.0S00</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="br_manually_groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>561</width>
      <height>95</height>
     </rect>
    </property>
    <property name="title">
     <string>Manually Brightness Control</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="mbr_label">
       <property name="text">
        <string>Brightness</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QSpinBox" name="mbr_spinBox">
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSlider" name="mbr_horizontalSlider">
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="mbr_Apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>br_l1</tabstop>
  <tabstop>br_l2</tabstop>
  <tabstop>br_l3</tabstop>
  <tabstop>br_l4</tabstop>
  <tabstop>br_l5</tabstop>
  <tabstop>br_l6</tabstop>
  <tabstop>br_l7</tabstop>
  <tabstop>br_l8</tabstop>
  <tabstop>auto_Apply</tabstop>
  <tabstop>mbr_spinBox</tabstop>
  <tabstop>mbr_horizontalSlider</tabstop>
  <tabstop>mbr_Apply</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mbr_spinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mbr_horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>172</x>
     <y>234</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mbr_horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>mbr_spinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>244</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
